### PR TITLE
Add recap blog post for SQL training 1

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _posts/2023-10-18-sql-training-1.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: "Getting started with Trino and SQL"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/trino-sql.png
+show_pagenav: false
+---
+
+In our training series [Learning SQL with Trino from the experts]({% post_url
+2023-09-27-training-series %}) Martin Traverso, Dain Sundstrom, David Phillips,
+and myself run through the wide range of SQL support and features of Trino with
```
Comment (colebow) [inline _posts/2023-10-18-sql-training-1.md:12]:
```suggestion
and myself will run through the wide range of SQL support and features of Trino with
```
Hasn't all happened yet! Let's make it clear it's still mostly upcoming.

File: _posts/2023-10-18-sql-training-1.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+our audience. In the first episode we covered the concepts of Trino and SQL, and
```
Comment (colebow) [inline _posts/2023-10-18-sql-training-1.md:13]:
```suggestion
our audience. In the first episode, we covered the concepts of Trino and SQL, and
```

File: _posts/2023-10-18-sql-training-1.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+then started to learn some basic SQL. Now you can take advantage of the
+recording and available resources to learn at your own pace.
+
+<!--more-->
+
+The recording of the event allows you to watch it all as if you attended live,
+jump to specific section as desired, or pause while you follow along with the
```
Comment (colebow) [inline _posts/2023-10-18-sql-training-1.md:20]:
```suggestion
jump to specific sections as desired, or pause while you follow along with the
```

File: _posts/2023-10-18-sql-training-1.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Following are a couple of specific timestamps for interesting
+topics snippets:
+
+* [What is Trino?](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=380)
+* [Catalogs and connectors](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=1163)
+* [Clients](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=1658)
+* [SQL WHERE statement](https://www.youtube.com/watch?v=SnvSBYhRZLg&t=3224)
+
+Find more on the description on YouTube.
```
Comment (colebow) [inline _posts/2023-10-18-sql-training-1.md:33]:
```suggestion
The full timestamps for every part of the talk are in the description on
YouTube, making it easy to skip to the sections that interest you.
```

File: _posts/2023-10-18-sql-training-1.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Also make sure you take advantage of these additional resources:
+
+* [General overview slide deck for the series]({{site.baserurl}}/assets/blog/sql-training-series-starburst-2023.pdf), with links to resources like our [community chat]({{site.baserurl}}/slack.html)
+* Slides deck for [SQL and Trino concepts](https://trinodb.github.io/presentations/presentations/sql-trino/index.html)
+* Slide deck for [SQL basics with Trino](https://trinodb.github.io/presentations/presentations/sql-basics/index.html), including a file with all SQL statements ready to go
+* [Trino: The Definitive Guide]({{site.baserurl}}/trino-the-definitive-guide.html)
+
+
+Now that you know of the series and saw the first iteration, make sure you
```
Comment (colebow) [inline _posts/2023-10-18-sql-training-1.md:43]:
```suggestion
Now that you know of the series and saw the first part of it, make sure you
```

**Review by colebow — CHANGES_REQUESTED** [review]
Just a few small changes to be made.

File: _posts/2023-10-18-sql-training-1.md (hunk 2 above)
Comment (mosabua) [reply _posts/2023-10-18-sql-training-1.md:13]: Done

File: _posts/2023-10-18-sql-training-1.md (hunk 1 above)
Comment (mosabua) [reply _posts/2023-10-18-sql-training-1.md:12]: Done

File: _posts/2023-10-18-sql-training-1.md (hunk 3 above)
Comment (mosabua) [reply _posts/2023-10-18-sql-training-1.md:20]: Done

File: _posts/2023-10-18-sql-training-1.md (hunk 4 above)
Comment (mosabua) [reply _posts/2023-10-18-sql-training-1.md:33]: Done

File: _posts/2023-10-18-sql-training-1.md (hunk 5 above)
Comment (mosabua) [reply _posts/2023-10-18-sql-training-1.md:43]: Done

**Review by colebow — APPROVED** [review]

**Review by martint — APPROVED** [review]

**Comment by mosabua** [comment]
Thank you @martint

**Pull request merged**